components/auth/GoogleAuth.tsx

```
@@ -18,16 +22,25 @@ import {
             await GoogleSignin.hasPlayServices()
-            const userInfo = await GoogleSignin.signIn()
-            if (userInfo.data && userInfo.data.idToken) {
-              const { data, error } = await supabase.auth.signInWithIdToken({
+            const response = await GoogleSignin.signIn()
+            if(isSuccessResponse(response)) {
+               const {idToken, user} = response.data
+
+               const { data, error } = await supabase.auth.signInWithIdToken({
                 provider: 'google',
-                token: userInfo.data.idToken,
+                token: idToken!!
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)